src/system/RoutesModule/Api/Base/CacheApi.php

```diff
@@ -14,14 +14,14 @@
 
 use ModUtil;
 use UserUtil;
-use Zikula_AbstractApi;
+use Zikula_AbstractBase;
 use Zikula_View;
 use Zikula_View_Theme;
 
 /**
  * Cache api base class.
  */
-class CacheApi extends Zikula_AbstractApi
+class CacheApi extends Zikula_AbstractBase
```

Comment:
This is subject of transition. Util classes also extend from AbstractBase. The difference between AbstractBase and AbstractApi is that AbstractApi is removed already.